docs/source/user-guide/tasks/build-packages/define-metadata.rst

```diff
@@ -618,22 +618,42 @@ package may not be testable, because the hashes will differ.
 
 .. _run_exports:
 
-Pin downstream
---------------
+Export runtime requirements
+---------------------------
 
-It is often useful to require a particular runtime library to be present at
-runtime when a particular library or package is used at build time. For example,
-if you use a C++ compiler to build a package with dynamic linkage, then it is
-likely that you also need to include the C++ runtime package corresponding to
-that compiler as a runtime requirement. Generally, these imposed pinnings should
-be added to the tool (compiler) or library (jpeg, bzip2, and so on) used at build
-time, rather than to the package using those tools or libraries.
+Some build or host :ref:`requirements` will impose a runtime requirement.
+Most commonly this is true for shared libraries (e.g. libpng), which are
+required for linking at build time, and for resolving the link at run time.
+With ``run_exports`` (new in conda-build 3) such a runtime requirement can be
+implicitly added by host requirements (e.g. libpng exports libpng), and with
+``run_exports/strong`` even by build requirements (e.g. gcc exports libgcc).
 
 .. code-block:: yaml
 
+   # meta.yaml of libpng
    build:
      run_exports:
-       - libstdc++
+       - libpng
+
+.. code-block:: yaml
+
+   # meta.yaml of gcc compiler
+   build:
+     run_exports:
+       strong:
+         - libgcc
+
+.. code-block:: yaml
+
+   # meta.yaml of some package using gcc and libpng
+   requirements:
+     build:
+       - gcc            # has a strong run export
+     host:
+       - libpng         # has a (weak) run export
+     run:
+       # - libgcc       <-- implicitly added
```

Comment:
Strong run exports also add their entry to the host section.  This mostly doesn't matter on Linux/Mac, but it is essential on Windows for getting the right libraries to link against, since windows is much more picky about compiler runtimes than linux/mac.

I can add a comment about this - I will push changes to your PR here.

Reply:
> Strong run exports also add their entry to the host section

That makes sense, thanks.